pkg/descheduler/strategies/duplicates_test.go

```diff
@@ -193,6 +221,13 @@ func TestFindDuplicatePods(t *testing.T) {
 		fakeClient.Fake.AddReactor("get", "nodes", func(action core.Action) (bool, runtime.Object, error) {
 			return true, node, nil
 		})
+		// //TODO how to pass replicaas as int in reactor action
+		// fakeClient.Fake.AddReactor("get", "replicationcontrollers", func(action core.Action) (bool, runtime.Object, error) {
+		// 	return true, node, nil
+		// })
+		// fakeClient.Fake.AddReactor("get", "replicaset", func(action core.Action) (bool, runtime.Object, error) {
+		// 	return true, nil, nil
+		// })
```

Comment:
Need help here. the reactor action expects an type that implements `DeepCopy`.
To return RS or RC, where do I define and implement the `DeepCopy` methods